draft-selander-ace-ake-authz.md

```diff
@@ -476,142 +477,36 @@ V receives the voucher response from W over the secure connection. If the receiv
 
 V verifies CRED_I and that U is an admissible device and then continues the EDHOC processing, or else discontinues the protocol.
 
-# ACE Profile
+# REST Interface at W
 
-The messages specified in this document may be carried between the endpoints in various protocols. This section defines an embedding as a profile of the ACE framework (see Appendix C of {{I-D.ietf-ace-oauth-authz}}).
+The interaction between V and W is enabled through a RESTful HTTPS interface exposed by W.
+V MUST perform a TLS handshake with W, reachable at the URI specified in LOC_W.
+V MUST use the TLS client authentication to authenticate to W, using the certificate containing the PK_V public key.
```

Comment:
I think we should be open for other ways to protect REST. Can we change to SHOULD and also mention secured CoAP?